### DISCUSSION TITLE
I came up with a github functionality extension Gitvideo

### DISCUSSION THREAD

**Discussion by Youthy96** [community/community#57321] [Programming Help]
### Select Topic Area

Show & Tell

### Body

Hi all.
I came up with a website.
Initially, it was conceived as an extension of the github functionality.
The idea is to make a video hosting on the interests of programming and technology without unnecessary information.
The main idea is that a person visits the site using Github and uploads a video showing their project, its work, programming approach and the whole process.
From his video, you can go to his Github account, you can also leave a link to the Gitvideo video in Github itself.

Here is the project [Gitvideo](https://gitvideo.pro/)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Youthy96 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
